acceptance/bundle/artifacts/shell/basic/output.txt

```diff
@@ -0,0 +1,7 @@
+
+>>> [CLI] bundle deploy
+Building my_artifact...
+Warn: my_artifact: Build succeeded
```

Comment:
Where does the warning come from?

Reply:
It comes from this line: https://github.com/databricks/cli/blob/75d72439b81d5248e707d0009ebd21b315508701/bundle/artifacts/build.go#L68

Seems like if we do not specify `type: whl` for the artifact this warning is emitted.